ocs_ci/utility/utils.py

```diff
@@ -2514,16 +2516,18 @@ def ceph_health_check(namespace=None, tries=20, delay=30):
         tries=tries,
         delay=delay,
         backoff=1,
-    )(ceph_health_check_base)(namespace)
+    )(ceph_health_check_base)(namespace, resolve_daemon_crash)
```

Comment:
I would go more generic like and name it like `fix_ceph_health` - I guess we might have multiple cases like this maybe in future we might have some fix for them.